### PR TITLE
python3Packages.mypy-boto3-builder: 4.12.0 -> 4.14.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 4.14.1

The `zeep` build failure is addressed in #126046.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 126162` at 5e10deb7 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/f9d00394dd65c45f26d88a20be97862e">awsebcli</a></li>
    <li><a href="https://gist.github.com/df31a1cc81bcf8d90cfbedfdb6313c8d">mlflow-server</a></li>
    <li><a href="https://gist.github.com/be8763cea998f2bb482c049b8c8de8b7">python38Packages.flower</a></li>
    <li><a href="https://gist.github.com/ad1c34d843c8b3b8fae5aa79b584f9bf">python38Packages.geopandas</a></li>
    <li><a href="https://gist.github.com/7e4d97813d5860279861fe28531fae42">python38Packages.osmnx</a></li>
    <li><a href="https://gist.github.com/ce3ba6c1cfc8e9d13711804ed6281153">python39Packages.flower</a></li>
    <li><a href="https://gist.github.com/d3cd72d7b0ac289c58f8794d814380c2">python39Packages.geopandas</a></li>
    <li><a href="https://gist.github.com/5f3e09989baf5f1fdc9d9514434371ca">python39Packages.osmnx</a></li>
  </ul>
</details>
<details>
  <summary>22 packages skipped due to time constraints:</summary>
  <ul>
    <li>deja-dup</li>
    <li>duplicity</li>
    <li>duply</li>
    <li>home-assistant</li>
    <li>lexicon</li>
    <li>python38Packages.rasterio</li>
    <li>python38Packages.scrapy</li>
    <li>python38Packages.scrapy-deltafetch</li>
    <li>python38Packages.scrapy-fake-useragent</li>
    <li>python38Packages.scrapy-splash</li>
    <li>...</li>
  </ul>
</details>
<details>
  <summary>124 packages built successfully:</summary>
  <ul>
    <li>aws-adfs (python38Packages.aws-adfs)</li>
    <li>aws-google-auth</li>
    <li>aws-mfa</li>
    <li>aws-sam-cli</li>
    <li>awscli</li>
    <li>awslogs</li>
    <li>barman</li>
    <li>bash-my-aws</li>
    <li>buildbot (python38Packages.buildbot)</li>
    <li>buildbot-full (python38Packages.buildbot-full)</li>
    <li>buildbot-ui (python38Packages.buildbot-ui)</li>
    <li>certbot-full</li>
    <li>checkov</li>
    <li>cloud-custodian</li>
    <li>credstash (python38Packages.credstash)</li>
    <li>dvc-with-remotes</li>
    <li>elasticsearch-curator</li>
    <li>git-remote-codecommit</li>
    <li>iceshelf</li>
    <li>luigi</li>
    <li>mapproxy</li>
    <li>nimbo</li>
    <li>nixops</li>
    <li>python38Packages.aiobotocore</li>
    <li>python38Packages.aws-sam-translator</li>
    <li>python38Packages.aws-xray-sdk</li>
    <li>python38Packages.boto3</li>
    <li>python38Packages.botocore</li>
    <li>python38Packages.cartopy</li>
    <li>python38Packages.celery</li>
    <li>python38Packages.certbot-dns-route53</li>
    <li>python38Packages.cfn-lint</li>
    <li>python38Packages.chalice</li>
    <li>python38Packages.dependency-injector</li>
    <li>python38Packages.django-anymail</li>
    <li>python38Packages.django-raster</li>
    <li>python38Packages.djmail</li>
    <li>python38Packages.ec2instanceconnectcli</li>
    <li>python38Packages.fiona</li>
    <li>python38Packages.flowlogs_reader</li>
    <li>python38Packages.fs-s3fs</li>
    <li>python38Packages.gensim</li>
    <li>python38Packages.gradient</li>
    <li>python38Packages.hass-nabucasa</li>
    <li>python38Packages.itemloaders</li>
    <li>python38Packages.kombu</li>
    <li>python38Packages.mapbox</li>
    <li>python38Packages.moto</li>
    <li>python38Packages.mypy-boto3-builder</li>
    <li>python38Packages.mypy-boto3-s3</li>
    <li>python38Packages.parsel</li>
    <li>python38Packages.pathy</li>
    <li>python38Packages.pycognito</li>
    <li>python38Packages.pynamodb</li>
    <li>python38Packages.s3fs</li>
    <li>python38Packages.s3transfer</li>
    <li>python38Packages.sagemaker</li>
    <li>python38Packages.sapi-python-client</li>
    <li>python38Packages.serverlessrepo</li>
    <li>python38Packages.slicedimage</li>
    <li>python38Packages.smart-open</li>
    <li>python38Packages.snowflake-connector-python</li>
    <li>python38Packages.snowflake-sqlalchemy</li>
    <li>python38Packages.tensorboardx</li>
    <li>python38Packages.warrant</li>
    <li>python39Packages.aiobotocore</li>
    <li>python39Packages.aws-adfs</li>
    <li>python39Packages.aws-sam-translator</li>
    <li>python39Packages.aws-xray-sdk</li>
    <li>python39Packages.boto3</li>
    <li>python39Packages.botocore</li>
    <li>python39Packages.buildbot</li>
    <li>python39Packages.buildbot-full</li>
    <li>python39Packages.buildbot-ui</li>
    <li>python39Packages.cartopy</li>
    <li>python39Packages.celery</li>
    <li>python39Packages.certbot-dns-route53</li>
    <li>python39Packages.cfn-lint</li>
    <li>python39Packages.chalice</li>
    <li>python39Packages.credstash</li>
    <li>python39Packages.dependency-injector</li>
    <li>python39Packages.django-anymail</li>
    <li>python39Packages.django-raster</li>
    <li>python39Packages.djmail</li>
    <li>python39Packages.ec2instanceconnectcli</li>
    <li>python39Packages.fiona</li>
    <li>python39Packages.flowlogs_reader</li>
    <li>python39Packages.fs-s3fs</li>
    <li>python39Packages.gradient</li>
    <li>python39Packages.hass-nabucasa</li>
    <li>python39Packages.itemloaders</li>
    <li>python39Packages.kombu</li>
    <li>python39Packages.mapbox</li>
    <li>python39Packages.moto</li>
    <li>python39Packages.mypy-boto3-builder</li>
    <li>python39Packages.mypy-boto3-s3</li>
    <li>python39Packages.parsel</li>
    <li>python39Packages.pycognito</li>
    <li>python39Packages.pynamodb</li>
    <li>python39Packages.s3fs</li>
    <li>python39Packages.s3transfer</li>
    <li>python39Packages.sagemaker</li>
    <li>python39Packages.sapi-python-client</li>
    <li>python39Packages.serverlessrepo</li>
    <li>python39Packages.slicedimage</li>
    <li>python39Packages.snowflake-connector-python</li>
    <li>python39Packages.snowflake-sqlalchemy</li>
    <li>python39Packages.warrant</li>
    <li>s3bro</li>
    <li>s4cmd</li>
    <li>snakemake</li>
    <li>sourcehut.buildsrht</li>
    <li>sourcehut.coresrht</li>
    <li>sourcehut.dispatchsrht</li>
    <li>sourcehut.gitsrht</li>
    <li>sourcehut.hgsrht</li>
    <li>sourcehut.hubsrht</li>
    <li>sourcehut.listssrht</li>
    <li>sourcehut.mansrht</li>
    <li>sourcehut.metasrht</li>
    <li>sourcehut.pastesrht</li>
    <li>sourcehut.todosrht</li>
    <li>ssm-session-manager-plugin</li>
    <li>streamlit</li>
  </ul>
</details>
<details>
  <summary>4 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/s3transfer/default.nix:46:3:
```
   |
46 |   meta = with lib; {
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/botocore/default.nix:43:3:
```
   |
43 |   meta = with lib; {
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/python-modules/boto3/default.nix:40:3:
```
   |
40 |   meta = {
   |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md">python-include-tests</a>

Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/s3transfer/default.nix:49:0:
```
   |
49 |     description = "A library for managing Amazon S3 transfers";
   | ^
```
</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 126162` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages marked as broken and skipped:</summary>
  <ul>
    <li>awsebcli</li>
    <li>mlflow-server</li>
    <li>python38Packages.flower</li>
    <li>python38Packages.geopandas</li>
    <li>python38Packages.osmnx</li>
    <li>python39Packages.flower</li>
    <li>python39Packages.geopandas</li>
    <li>python39Packages.osmnx</li>
  </ul>
</details>
<details>
  <summary>9 packages failed to build:</summary>
  <ul>
    <li>lexicon</li>
    <li>python38Packages.scrapy</li>
    <li>python38Packages.scrapy-deltafetch</li>
    <li>python38Packages.scrapy-fake-useragent</li>
    <li>python38Packages.scrapy-splash</li>
    <li>python39Packages.scrapy</li>
    <li>python39Packages.scrapy-deltafetch</li>
    <li>python39Packages.scrapy-fake-useragent</li>
    <li>python39Packages.scrapy-splash</li>
  </ul>
</details>
<details>
  <summary>137 packages built:</summary>
  <ul>
    <li>aws-adfs (python38Packages.aws-adfs)</li>
    <li>aws-google-auth</li>
    <li>aws-mfa</li>
    <li>aws-sam-cli</li>
    <li>awscli</li>
    <li>awslogs</li>
    <li>barman</li>
    <li>bash-my-aws</li>
    <li>buildbot (python38Packages.buildbot)</li>
    <li>buildbot-full (python38Packages.buildbot-full)</li>
    <li>buildbot-ui (python38Packages.buildbot-ui)</li>
    <li>certbot-full</li>
    <li>checkov</li>
    <li>cloud-custodian</li>
    <li>credstash (python38Packages.credstash)</li>
    <li>deja-dup</li>
    <li>duplicity</li>
    <li>duply</li>
    <li>dvc-with-remotes</li>
    <li>elasticsearch-curator</li>
    <li>git-remote-codecommit</li>
    <li>home-assistant</li>
    <li>iceshelf</li>
    <li>luigi</li>
    <li>mapproxy</li>
    <li>nimbo</li>
    <li>nixops</li>
    <li>python38Packages.aiobotocore</li>
    <li>python38Packages.aws-sam-translator</li>
    <li>python38Packages.aws-xray-sdk</li>
    <li>python38Packages.boto3</li>
    <li>python38Packages.botocore</li>
    <li>python38Packages.cartopy</li>
    <li>python38Packages.celery</li>
    <li>python38Packages.certbot-dns-route53</li>
    <li>python38Packages.cfn-lint</li>
    <li>python38Packages.chalice</li>
    <li>python38Packages.dependency-injector</li>
    <li>python38Packages.django-anymail</li>
    <li>python38Packages.django-raster</li>
    <li>python38Packages.djmail</li>
    <li>python38Packages.ec2instanceconnectcli</li>
    <li>python38Packages.fiona</li>
    <li>python38Packages.flowlogs_reader</li>
    <li>python38Packages.fs-s3fs</li>
    <li>python38Packages.gensim</li>
    <li>python38Packages.gradient</li>
    <li>python38Packages.hass-nabucasa</li>
    <li>python38Packages.itemloaders</li>
    <li>python38Packages.kombu</li>
    <li>python38Packages.mapbox</li>
    <li>python38Packages.moto</li>
    <li>python38Packages.mypy-boto3-builder</li>
    <li>python38Packages.mypy-boto3-s3</li>
    <li>python38Packages.parsel</li>
    <li>python38Packages.pathy</li>
    <li>python38Packages.pycognito</li>
    <li>python38Packages.pynamodb</li>
    <li>python38Packages.rasterio</li>
    <li>python38Packages.s3fs</li>
    <li>python38Packages.s3transfer</li>
    <li>python38Packages.sagemaker</li>
    <li>python38Packages.sapi-python-client</li>
    <li>python38Packages.serverlessrepo</li>
    <li>python38Packages.slack-sdk</li>
    <li>python38Packages.slicedimage</li>
    <li>python38Packages.smart-open</li>
    <li>python38Packages.snowflake-connector-python</li>
    <li>python38Packages.snowflake-sqlalchemy</li>
    <li>python38Packages.spacy</li>
    <li>python38Packages.spacy-transformers</li>
    <li>python38Packages.tensorboardx</li>
    <li>python38Packages.textacy</li>
    <li>python38Packages.warrant</li>
    <li>python39Packages.aiobotocore</li>
    <li>python39Packages.aws-adfs</li>
    <li>python39Packages.aws-sam-translator</li>
    <li>python39Packages.aws-xray-sdk</li>
    <li>python39Packages.boto3</li>
    <li>python39Packages.botocore</li>
    <li>python39Packages.buildbot</li>
    <li>python39Packages.buildbot-full</li>
    <li>python39Packages.buildbot-ui</li>
    <li>python39Packages.cartopy</li>
    <li>python39Packages.celery</li>
    <li>python39Packages.certbot-dns-route53</li>
    <li>python39Packages.cfn-lint</li>
    <li>python39Packages.chalice</li>
    <li>python39Packages.credstash</li>
    <li>python39Packages.dependency-injector</li>
    <li>python39Packages.django-anymail</li>
    <li>python39Packages.django-raster</li>
    <li>python39Packages.djmail</li>
    <li>python39Packages.ec2instanceconnectcli</li>
    <li>python39Packages.fiona</li>
    <li>python39Packages.flowlogs_reader</li>
    <li>python39Packages.fs-s3fs</li>
    <li>python39Packages.gradient</li>
    <li>python39Packages.hass-nabucasa</li>
    <li>python39Packages.itemloaders</li>
    <li>python39Packages.kombu</li>
    <li>python39Packages.mapbox</li>
    <li>python39Packages.moto</li>
    <li>python39Packages.mypy-boto3-builder</li>
    <li>python39Packages.mypy-boto3-s3</li>
    <li>python39Packages.parsel</li>
    <li>python39Packages.pycognito</li>
    <li>python39Packages.pynamodb</li>
    <li>python39Packages.rasterio</li>
    <li>python39Packages.s3fs</li>
    <li>python39Packages.s3transfer</li>
    <li>python39Packages.sagemaker</li>
    <li>python39Packages.sapi-python-client</li>
    <li>python39Packages.serverlessrepo</li>
    <li>python39Packages.slack-sdk</li>
    <li>python39Packages.slicedimage</li>
    <li>python39Packages.snowflake-connector-python</li>
    <li>python39Packages.snowflake-sqlalchemy</li>
    <li>python39Packages.tensorboardx</li>
    <li>python39Packages.warrant</li>
    <li>s3bro</li>
    <li>s4cmd</li>
    <li>snakemake</li>
    <li>sourcehut.buildsrht</li>
    <li>sourcehut.coresrht</li>
    <li>sourcehut.dispatchsrht</li>
    <li>sourcehut.gitsrht</li>
    <li>sourcehut.hgsrht</li>
    <li>sourcehut.hubsrht</li>
    <li>sourcehut.listssrht</li>
    <li>sourcehut.mansrht</li>
    <li>sourcehut.metasrht</li>
    <li>sourcehut.pastesrht</li>
    <li>sourcehut.todosrht</li>
    <li>ssm-session-manager-plugin</li>
    <li>streamlit</li>
    <li>tts</li>
  </ul>
</details>

**Review by SuperSandro2000 — APPROVED** [review]
diff LGTM

File: pkgs/development/python-modules/scrapy/default.nix [hunk 1]
```diff
@@ -1,64 +1,74 @@
...
-, botocore
-, itemadapter
-, itemloaders
 }:
 
 buildPythonPackage rec {
-  version = "2.4.1";
-  pname = "Scrapy";
+  pname = "scrapy";
+  # 2.5.0 require h2<4
```
Comment (marsam) [inline pkgs/development/python-modules/scrapy/default.nix:33]: Would you mind checking whether we can apply https://github.com/scrapy/scrapy/commit/c5b1ee810167266fcd259f263dbfc0fe0204761a?

Comment (fabaff) [reply pkgs/development/python-modules/scrapy/default.nix]: Seems to work if combined with another patch.

**Review by marsam — APPROVED** [review]
LGTM. Thank you!

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 126162` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages marked as broken and skipped:</summary>
  <ul>
    <li>awsebcli</li>
    <li>mlflow-server</li>
    <li>python38Packages.flower</li>
    <li>python38Packages.geopandas</li>
    <li>python38Packages.osmnx</li>
    <li>python39Packages.flower</li>
    <li>python39Packages.geopandas</li>
    <li>python39Packages.osmnx</li>
  </ul>
</details>
<details>
  <summary>146 packages built:</summary>
  <ul>
    <li>aws-adfs (python38Packages.aws-adfs)</li>
    <li>aws-google-auth</li>
    <li>aws-mfa</li>
    <li>aws-sam-cli</li>
    <li>awscli</li>
    <li>awslogs</li>
    <li>barman</li>
    <li>bash-my-aws</li>
    <li>buildbot (python38Packages.buildbot)</li>
    <li>buildbot-full (python38Packages.buildbot-full)</li>
    <li>buildbot-ui (python38Packages.buildbot-ui)</li>
    <li>certbot-full</li>
    <li>checkov</li>
    <li>cloud-custodian</li>
    <li>credstash (python38Packages.credstash)</li>
    <li>deja-dup</li>
    <li>duplicity</li>
    <li>duply</li>
    <li>dvc-with-remotes</li>
    <li>elasticsearch-curator</li>
    <li>git-remote-codecommit</li>
    <li>home-assistant</li>
    <li>iceshelf</li>
    <li>lexicon</li>
    <li>luigi</li>
    <li>mapproxy</li>
    <li>nimbo</li>
    <li>nixops</li>
    <li>python38Packages.aiobotocore</li>
    <li>python38Packages.aws-sam-translator</li>
    <li>python38Packages.aws-xray-sdk</li>
    <li>python38Packages.boto3</li>
    <li>python38Packages.botocore</li>
    <li>python38Packages.cartopy</li>
    <li>python38Packages.celery</li>
    <li>python38Packages.certbot-dns-route53</li>
    <li>python38Packages.cfn-lint</li>
    <li>python38Packages.chalice</li>
    <li>python38Packages.dependency-injector</li>
    <li>python38Packages.django-anymail</li>
    <li>python38Packages.django-raster</li>
    <li>python38Packages.djmail</li>
    <li>python38Packages.ec2instanceconnectcli</li>
    <li>python38Packages.fiona</li>
    <li>python38Packages.flowlogs_reader</li>
    <li>python38Packages.fs-s3fs</li>
    <li>python38Packages.gensim</li>
    <li>python38Packages.gradient</li>
    <li>python38Packages.hass-nabucasa</li>
    <li>python38Packages.itemloaders</li>
    <li>python38Packages.kombu</li>
    <li>python38Packages.mapbox</li>
    <li>python38Packages.moto</li>
    <li>python38Packages.mypy-boto3-builder</li>
    <li>python38Packages.mypy-boto3-s3</li>
    <li>python38Packages.parsel</li>
    <li>python38Packages.pathy</li>
    <li>python38Packages.pycognito</li>
    <li>python38Packages.pynamodb</li>
    <li>python38Packages.rasterio</li>
    <li>python38Packages.s3fs</li>
    <li>python38Packages.s3transfer</li>
    <li>python38Packages.sagemaker</li>
    <li>python38Packages.sapi-python-client</li>
    <li>python38Packages.scrapy</li>
    <li>python38Packages.scrapy-deltafetch</li>
    <li>python38Packages.scrapy-fake-useragent</li>
    <li>python38Packages.scrapy-splash</li>
    <li>python38Packages.serverlessrepo</li>
    <li>python38Packages.slack-sdk</li>
    <li>python38Packages.slicedimage</li>
    <li>python38Packages.smart-open</li>
    <li>python38Packages.snowflake-connector-python</li>
    <li>python38Packages.snowflake-sqlalchemy</li>
    <li>python38Packages.spacy</li>
    <li>python38Packages.spacy-transformers</li>
    <li>python38Packages.tensorboardx</li>
    <li>python38Packages.textacy</li>
    <li>python38Packages.warrant</li>
    <li>python39Packages.aiobotocore</li>
    <li>python39Packages.aws-adfs</li>
    <li>python39Packages.aws-sam-translator</li>
    <li>python39Packages.aws-xray-sdk</li>
    <li>python39Packages.boto3</li>
    <li>python39Packages.botocore</li>
    <li>python39Packages.buildbot</li>
    <li>python39Packages.buildbot-full</li>
    <li>python39Packages.buildbot-ui</li>
    <li>python39Packages.cartopy</li>
    <li>python39Packages.celery</li>
    <li>python39Packages.certbot-dns-route53</li>
    <li>python39Packages.cfn-lint</li>
    <li>python39Packages.chalice</li>
    <li>python39Packages.credstash</li>
    <li>python39Packages.dependency-injector</li>
    <li>python39Packages.django-anymail</li>
    <li>python39Packages.django-raster</li>
    <li>python39Packages.djmail</li>
    <li>python39Packages.ec2instanceconnectcli</li>
    <li>python39Packages.fiona</li>
    <li>python39Packages.flowlogs_reader</li>
    <li>python39Packages.fs-s3fs</li>
    <li>python39Packages.gradient</li>
    <li>python39Packages.hass-nabucasa</li>
    <li>python39Packages.itemloaders</li>
    <li>python39Packages.kombu</li>
    <li>python39Packages.mapbox</li>
    <li>python39Packages.moto</li>
    <li>python39Packages.mypy-boto3-builder</li>
    <li>python39Packages.mypy-boto3-s3</li>
    <li>python39Packages.parsel</li>
    <li>python39Packages.pycognito</li>
    <li>python39Packages.pynamodb</li>
    <li>python39Packages.rasterio</li>
    <li>python39Packages.s3fs</li>
    <li>python39Packages.s3transfer</li>
    <li>python39Packages.sagemaker</li>
    <li>python39Packages.sapi-python-client</li>
    <li>python39Packages.scrapy</li>
    <li>python39Packages.scrapy-deltafetch</li>
    <li>python39Packages.scrapy-fake-useragent</li>
    <li>python39Packages.scrapy-splash</li>
    <li>python39Packages.serverlessrepo</li>
    <li>python39Packages.slack-sdk</li>
    <li>python39Packages.slicedimage</li>
    <li>python39Packages.snowflake-connector-python</li>
    <li>python39Packages.snowflake-sqlalchemy</li>
    <li>python39Packages.tensorboardx</li>
    <li>python39Packages.warrant</li>
    <li>s3bro</li>
    <li>s4cmd</li>
    <li>snakemake</li>
    <li>sourcehut.buildsrht</li>
    <li>sourcehut.coresrht</li>
    <li>sourcehut.dispatchsrht</li>
    <li>sourcehut.gitsrht</li>
    <li>sourcehut.hgsrht</li>
    <li>sourcehut.hubsrht</li>
    <li>sourcehut.listssrht</li>
    <li>sourcehut.mansrht</li>
    <li>sourcehut.metasrht</li>
    <li>sourcehut.pastesrht</li>
    <li>sourcehut.todosrht</li>
    <li>ssm-session-manager-plugin</li>
    <li>streamlit</li>
    <li>tts</li>
  </ul>
</details>

**Pull request merged**